config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml

```diff
@@ -25,32 +20,20 @@ spec:
   nodes:
       edpm-compute-0:
         hostName: edpm-compute-0
-        ansible:
-          ansibleHost: 192.168.122.100
-          ansibleVars:
-            ctlplane_ip: 192.168.122.100
-            internal_api_ip: 172.17.0.100
-            storage_ip: 172.18.0.100
-            tenant_ip: 172.19.0.100
-            fqdn_internal_api: edpm-compute-0.example.com
-  networkAttachments:
-    - ctlplane
   nodeTemplate:
+    networks:
+      - name: CtlPlane
+        subnetName: subnet1
+        defaultRoute: true
+      - name: InternalApi
+        subnetName: subnet1
+      - name: Storage
+        subnetName: subnet1
+      - name: Tenant
+        subnetName: subnet1
```

Comment:
If we use IPAM, we can eliminate all of the unnecessary `_ip` `_cidr` type Ansible vars below. This is probably advantageous as each of those presents a new opportunity to make a mistake. Using IPAM ensures they are generated automagically for you.

IMO, we should use IPAM for all of the samples for that reason. But happy to discuss

Reply:
I agree, let's use IPAM for all the samples, other than a single sample that doesn't use IPAM and shows just that deviation.